packages/create/templates/demo/package.json

```diff
@@ -11,11 +11,11 @@
 	"devDependencies": {
 		"@fontsource/fira-mono": "^5.0.0",
 		"@neoconfetti/svelte": "^2.0.0",
-		"@sveltejs/adapter-auto": "^3",
-		"@sveltejs/kit": "^2",
-		"@sveltejs/vite-plugin-svelte": "^4.0.0",
+		"@sveltejs/adapter-auto": "^4.0.0",
+		"@sveltejs/kit": "^2.9.0",
```

Comment:
I need 2.16.0 in https://github.com/sveltejs/cli/pull/409. We should probably make it the same here to avoid merge conflicts